dipy_web/urls.py

```diff
@@ -20,3 +20,6 @@
     url(r'^admin/', admin.site.urls),
     url(r'^', include('website.urls')),
 ]
+
+handler404 = 'website.views.custom404'
```

Comment:
For my info, those do not seem to be added to url_patterns. Are those names (handler404 and 500) reserved words?